include/oboe/Definitions.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2016 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef OBOE_DEFINITIONS_H
+#define OBOE_DEFINITIONS_H
+
+#include <cstdint>
+#include <aaudio/AAudio.h>
+
+namespace oboe {
+
+    constexpr int32_t kUnspecified = 0;
+
+    constexpr int64_t kNanosPerMicrosecond =    1000;
```

Comment:
Please add a TODO to consider using std::chrono, as it provides a lot of time unit conversion utilities.

Reply:
Done